tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -126,3 +138,23 @@ Verify Cuda Builds Are Completed
         END
         Should Be Equal As Strings    ${pre}[3]    Complete
     END
+
+Verify Link And Run Commands On Prometheus Pod For Jupyterhub And Rhods-Dashboard URLs
+    [Documentation]    Verifies the links and Run curl command on prometheus pod
+    [Arguments]    ${links}
+    ${jupyterhub} =    Set Variable    jupyterhub
+    ${rhods_dashboard} =    Set Variable    rhods-dashboard
+    ${pod_name} =    Find First Pod By Name    namespace=redhat-ods-monitoring    pod_start_with=prometheus-
+    FOR    ${link}    IN    @{links}
+        IF    "${jupyterhub}" in "${link}"
```

Comment:
Why do you need to check if jupyterhub or rhods_dashboard is in the link?

Also, you don't need the variables, you can just do  `IF    "jupyterhub" in "${link}"`  no?

Reply:
In polarion `Verify that there are two targets under "user_facing_endpoints_status" (at the bottom), one for jupyterhub and another for rhods-dashboard.` mentioned that so that's why I am verifying the links are correct..